src/HashBus.Viewer.MostMentioned/LeaderboardView.cs

```diff
@@ -124,23 +124,29 @@ static class LeaderboardView<TEntry> where TEntry : WebApi.IEntry
                     ColorConsole.WriteLine(new ColorToken[] { padding }.Concat(line).ToArray());
                 }
 
-                ColorConsole.WriteLine(
-                    padding,
-                    $"Total {itemsName}:".Gray(),
-                    " ",
-                    $"{currentLeaderboard?.Count ?? 0:N0}"
-                        .Color(currentLeaderboard?.Count - previousLeaderboard?.Count > 0 ? movementColors[-1] : movementColors[0]),
-                    $" since ".Gray(),
-                    $"{currentLeaderboard.Since?.ToLocalTime().ToString() ?? ("started")}".Gray());
+                if (currentLeaderboard.Since.HasValue)
+                {
+                    ColorConsole.WriteLine(
+                        padding,
+                        $"{currentLeaderboard.Since?.ToLocalTime().ToString()} to {currentLeaderboard.LastActivityDateTime?.ToLocalTime().ToString()}".Gray());

```

Comment:
We can remove the two `ToString()` calls in this line.

The call to `ToString()` in the original version was only present because of the null coalesce with `"started"`.